docs(Quickspin): Removed install and added how to in index. Commented out tutorial and… (#1592)
armory/docs: content/en/armory-enterprise/quick-spin-instructions/_index.md

no_list: true
weight: 3
description: >

>   Thank You for signing up to Quick Spin. Use these instructions to evaluate Armory's Continuous Deployment Self-Hosted solution, or get help from our team [Contact Us](https://www.armory.io/contact-us/).

Reviewer: Hugo converts the `description` front matter field meta tags. Then content of this field is very important for SEO and appears in search engine results so the description should concisely describe the page content.
```

  <meta name="description" content="Thank You for signing up to Quick Spin. Use these instructions to evaluate Armory&#39;s Continuous Deployment Self-Hosted solution, or get help from our team [Contact Us](https://www.armory.io/contact-us/).
">
<meta property="og:title" content="Quick Spin" />
<meta property="og:description" content="Thank You for signing up to Quick Spin. Use these instructions to evaluate Armory&#39;s Continuous Deployment Self-Hosted solution, or get help from our team [Contact Us](https://www.armory.io/contact-us/).
" />
<meta property="og:type" content="website" />
<meta property="og:url" content="/armory-enterprise/quick-spin-instructions/" />
<meta property="og:site_name" content="Armory Docs" />
<meta itemprop="name" content="Quick Spin">
<meta itemprop="description" content="Thank You for signing up to Quick Spin. Use these instructions to evaluate Armory&#39;s Continuous Deployment Self-Hosted solution, or get help from our team [Contact Us](https://www.armory.io/contact-us/).
"><meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Quick Spin"/>
<meta name="twitter:description" content="Thank You for signing up to Quick Spin. Use these instructions to evaluate Armory&#39;s Continuous Deployment Self-Hosted solution, or get help from our team [Contact Us](https://www.armory.io/contact-us/).
"/>
```

FWIW the URL must also follow  SEO guidelines, so `/quick-spin/` is preferable as a section 

So the description could be something like "Learn how to use Armory Quick Spin to evaluate the Armory Continuous Deployment Self-Hosted platform."

exclude_search: true
toc_hide: true
---

> The Quick Spin container is not intended for production use.

> {{% heading "prereq" %}}

Reviewer: ```suggestion
## {{% heading "prereq" %}}
```

> To run the Quick `quick-spin` container you need a host machine that can support a running instance with 

Reviewer: ```suggestion
To run the `quick-spin` container,  you need a host machine that can support a running instance with: 
```

  - 4 vCPU
  - 8GB Memory
  - 2GB Disk space

>   - The host machine must have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed. Docker Compose is installed with Docker Desktop.

Reviewer: This bullet point doesn't match the preceding three, so how about moving it to its own sentence? and perhaps placing it at the beginning of the section? then go on to mention the requirements for running the container

> ## Installing Quick Spin

Reviewer: ```suggestion
## Install Quick Spin
```
No gerunds in headings (to match most of the rest of the docs content headings. we switched from gerunds to active form of the verb a couple of years ago)

> Open a terminal to the preferred installation directory and run the following docker command:

Reviewer: ```suggestion
Open a terminal to the preferred installation directory and run the following Docker command:
```

```shell

>  docker compose pull quickspin armory/quickspin:latest

Reviewer: The Docker image is `amory/quick-spin`
https://hub.docker.com/r/armory/quick-spin

Reviewer: Also, the command is `docker pull armory/quick-spin`

`docker compose pull quickspin armory/quick-spin:latest` throws a `no configuration file provided: not found` error

Reviewer: And then you have to run the container
`docker run --name quick-spin --rm -p 9000:80 -t armory/quick-spin:latest`

Reviewer: the commands for pulling and running the image are on the dockerhub page

Reviewer: After listing the commands to pull and run, how about adding the success message displayed in the terminal?

Something like

Upon success, you should see a message like:
```
```bash
 Waiting for Quick-Spin to become ready . . .
    +---------------------------------+
    |                                 |
    | Quick-Spin is ready to go!      |
    |                                 |
    | Go to: http://localhost:9000    |
    |                                 |
    | To stop Quick-Spin press ctrl+c |
    |                                 |
    +---------------------------------+
```
```

```

> {{< alert color="success" >}}Quick Spin is running on `localhost:9000`. 
> The provided instance deploys a basic NGINX instance and sets up a minimal deployment configuration. Now you can start using Spinnaker to deploy your own applications.{{< /alert >}}

Reviewer: ```suggestion
{{< alert color="success" >}}You can access the UI on `localhost:9000`. 
Quick Spin deploys a basic NGINX instance and sets up a minimal deployment configuration. Now you can start using Armory Continuous Deployment (Armory CD) to deploy your own applications.{{< /alert >}}
```

> ## What can I do with Quick Spin?

Reviewer: ```suggestion
## What you can do with Quick Spin
```

Wondering why this isn't the first section in the page since it introduces the product??

Quick Spin is a containerized Armory CD instance that you install with a single shell command. You can use this minimal instance to evaluate the Armory CD Self-Hosted product.

> Once you have Quick Spin running, you can use it to evaluate Armory CD Self Hosted and to learn how to use Spinnaker. You can also use Quick Spin to test out Spinnaker features and to learn how to configure Spinnaker. 

Reviewer: L35 is saying almost the same thing as L33, so how about omitting L 35 and adding a bit to L33?
Something like

```
Quick Spin is a containerized Armory CD instance that you install with a single shell command. You can use this minimal instance to evaluate the Armory CD Self-Hosted product in functional areas such as configuration and deploying an application.
```

> Included with the Quick Spin container is a sample application and three sample pipelines that you can use to test out Spinnaker. After installing Quick Spin, you can use the sample application and pipelines to test out Spinnaker features.

Reviewer: ```suggestion
Quick Spin comes configured with a sample application and three pipelines that you can use to test Armory CD features.
```

> ## Using Quick Spin

Reviewer: ```suggestion
## Use Quick Spin
```

1. Click *Save Changes*.

> 1. Select *Pipelines* for the left navigation menu and click *Start Manual Execution* for the `prepare-quick-spin-environment` pipeline.

Reviewer: [Use bold for user interface elements](https://docs.armory.io/contribute/create/guides/docs-style-guide/#use-bold-for-user-interface-elements)

{{< alert color="success" >}}Your pipeline is deployed. When it completes you can try out the `basic-deployment` pipeline. 
{{< /alert >}}

> - To learn more about each pipeline, see the [Quick Spin Pipelines](/armory-enterprise/quickspin/pipelines/) page.

Reviewer: ```suggestion
- To learn more about each pipeline, see the [Quick Spin Pipelines]({{< ref "/armory-enterprise/quickspin/pipelines/" >}}) page.
```
Please no hard-coded URLs ;-p. and also this link produces a 404 because it has /quickspin/

> - Use the `teardown-quick-spin environment` pipeline to remove the a deployment.

Reviewer: ```suggestion
- Use the `teardown-quick-spin environment` pipeline to remove a deployment.
```
minor typo

> ### Quick Spin limitations

Reviewer: l think this section is important enough to be a `##`  - thoughts?

> The Quick Spin Spinnaker instance is configured for easy install and evaluation, testing, or POC use. It has the following usage limitations:

Reviewer: ```suggestion
Quick Spin is designed for evaluation, testing, or POC use. It has the following usage limitations:
```
